.claude/skills/github-pr-creation/SKILL.md

```
@@ -0,0 +1,116 @@
+git pull origin develop
+
+# 브랜치 생성
+git checkout -b feature/[기능명-kebab-case]
+# 또는
+git checkout -b fix/sonarcloud-quality-$(date +%Y%m%d)
+```
+
+## 커밋 규칙
+
+컨벤셔널 커밋 형식:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)